unit_testing/test_centerline.py

```diff
@@ -70,16 +71,20 @@
 ]
 
 param_optic = [
-    ({'fname_image': 'sct_testing_data/t2/t2.nii.gz',
+    ({'fname_image':
+        sct_dir_local_path('sct_testing_data/t2/t2.nii.gz'),
```

Comment:
i'm just worried that if, at some point, we decide to download the testing data somewhere else (e.g. ~/), this will break. But maybe we can ignore that potential issue for now

Reply:
actually, when running pytest on my local station, i always do it in a temp folder (outside of SCT dir), so this *will* be a problem. Ideally we should set an env variable for where the testing data are downloaded

Reply:
Sure thing, I might keep `sct_dir_local_path` but we could have another utility fun that fetches the test data path from an environment variable and join subpaths.

Reply:
I have now added `sct_test_path` which wraps an environment lookup for `SCT_TESTING_DATA` (default = "") and joins `sct_testing_data` along with any additional paths